src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -226,6 +226,9 @@
   <data name="net_http_unix_https_support_unavailable_libcurl" xml:space="preserve">
     <value>The libcurl library in use ({0}) does not support HTTPS.</value>
   </data>
+  <data name="net_http_libcurl_newer_openssl_notsupported" xml:space="preserve">
+    <value>The libcurl library in use is built with {0}, we require version OpenSSL/1.0</value>
+  </data>
```

Comment:
I think the options at the bottom of the file cover this (https://github.com/Priya91/corefx-1/blob/abad3ab0b6b7f7bb8bb847b1cf29667e7c76dfe3/src/System.Net.Http/src/Resources/Strings.resx#L298-L306); I don't think we need to mention an OpenSSL version by name here.  If we need to, it should be a format parameter, so it gets updated with the code.